src/main/java/uk/ac/ebi/embl/gff3tools/validation/builtin/DuplicateSeqIdValidation.java

```diff
@@ -13,24 +13,16 @@
 import java.util.HashSet;
 import uk.ac.ebi.embl.gff3tools.exception.*;
 import uk.ac.ebi.embl.gff3tools.exception.ValidationException;
-import uk.ac.ebi.embl.gff3tools.gff3.GFF3Annotation;
 import uk.ac.ebi.embl.gff3tools.gff3.GFF3Feature;
-import uk.ac.ebi.embl.gff3tools.validation.AnnotationValidation;
-import uk.ac.ebi.embl.gff3tools.validation.FeatureValidation;
+import uk.ac.ebi.embl.gff3tools.validation.*;
 
-public class DuplicateSeqIdValidation implements FeatureValidation, AnnotationValidation {
-
-    public static final String VALIDATION_RULE = "GFF3_DUPLICATE_SEQID";
+@Gff3Validation
+public class DuplicateSeqIdValidation extends Validation {
 
     private HashSet<String> processedAnnotations = new HashSet<>();
     private String currentAccession = null;
 
-    @Override
-    public String getValidationRule() {
-        return VALIDATION_RULE;
-    }
-
-    @Override
+    @ValidationMethod(type = ValidationType.FEATURE)
```

Comment:
We need to name the validations. What happens if we do not name it? What does this mean?

Reply:
If it is not named, we cannot apply any rules to this. these are for rules that must always execute